packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -0,0 +1,52 @@
+const { Command } = require('@oclif/command');
+
+const { startSpinner, endSpinner } = require('../utils/display');
+
+const inquirer = require('inquirer');
+
+class ZapierBaseCommand extends Command {
+  run() {
+    this.parseFlags();
+    return this.perform();
+  }
+
+  parseFlags() {
+    // normally this is called via `this.parse(SomeCommand)`, but that's error-prone and I got tired of typing it
+    // .constructor is the static class
+    const { flags, args } = this.parse(Object.getPrototypeOf(this).constructor);
```

Comment:
sure! 

This is how all the docs say we should parse args:)

```js
// SomeCommand.js

class SomeCommand extends BaseCommand {
  async run() {
    const { flags } = this.parse(SomeCommand);
  }
}

// SomeOtherCommand.js

class SomeOtherCommand extends BaseCommand {
  async run() {
    const { flags } = this.parse(SomeOtherCommand);
  }
}
```

That's not too bad, but if you forget that line (like if you're writing a new command from scratch rather than pasting an existing one, or you're writing a command with no args) then none of the arg parsing happens. That includes `-h` and `--version`, which we'd otherwise consider "root" flags.

I was writing this from the perspective of "how can we make this as easy as possible for future devs to add new command without having to read a lot of existing code". So rather than have it be possible to not parse args at all, I figure we put it in a base class and make sure it always happens. Even as we're writing these "new" commands, we won't have to type/paste it every time.